### PR TITLE
Distribute 3D hybrid simulations.

### PR DESCRIPTION
**Pull request by sriharshakandala**
Distribute 3D hybrid simulations.

### CODE REVIEW

File: examples/hybrid/dbubble3d_invariant_totalenergy.jl [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+import Plots
+Plots.GRBackend()
+
+dir = "bubble3d_invariant_etot"
+path = joinpath(@__DIR__, "output", dir)
+mkpath(path)
+
+#=
+# post-processing
+Es = [sum(Y.Yc.ρe) for u in sol_invariant.u]
```
Comment (sriharshakandala) [inline examples/hybrid/dbubble3d_invariant_totalenergy.jl:348]: @akshaysridhar : Should this be 
```
Es = [sum(u.Yc.ρe) for u in sol_invariant.u]
````

Comment (akshaysridhar) [reply examples/hybrid/dbubble3d_invariant_totalenergy.jl:348]: Yes. Likewise in the corresponding 2D example outputs. Thanks @sriharshakandala

File: examples/hybrid/dbubble3d_invariant_totalenergy.jl [hunk 2]
```diff
@@ -0,0 +1,379 @@
+push!(LOAD_PATH, joinpath(@__DIR__, "..", ".."))
```
Comment (simonbyrne) [inline examples/hybrid/dbubble3d_invariant_totalenergy.jl:1]: this shouldn't be needed now.

File: examples/bickleyjet/bickleyjet_cg_invariant_hypervisc.jl [hunk 3]
```diff
@@ -12,7 +12,7 @@ import ClimaCore:
 using OrdinaryDiffEq: ODEProblem, solve, SSPRK33
 
 using Logging
-#ENV["CLIMACORE_DISTRIBUTED"] = "MPI"
+ENV["CLIMACORE_DISTRIBUTED"] = "MPI"
```
Comment (simonbyrne) [inline examples/bickleyjet/bickleyjet_cg_invariant_hypervisc.jl:15]: this should be set externally

**Comment by sriharshakandala** [comment]
bors try

**Comment by sriharshakandala** [comment]
bors try

**Pull request merged**